content/ja/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -85,7 +85,7 @@ <h2 style="color: #3771e3;">Kubenretes上にはじめてのアプリケーショ
         <div class="row">
             <div class="col-md-8">
 
-                <p>最初のDeploymentには、Dockerコンテナにパッケージされた<a href="https://nodejs.org">Node.js</a>アプリケーションを使用します。Node.jsアプリケーションを作成してDockerコンテナをデプロイするには、<a href="/ja/docs/tutorials/hello-minikube/#create-your-node-js-application">Hello Minikubeチュートリアル</a>の指示に従ってください。</p>
+                <p>最初のDeploymentには、DockerコンテナにパッケージされたNode.jsアプリケーションを使用します。Node.jsアプリケーションを作成してDockerコンテナをデプロイするには、<a href="/docs/tutorials/hello-minikube/">Hello Minikubeチュートリアル</a>の指示に従ってください。</p>
```

Comment:
```suggestion
                <p>最初のDeploymentには、DockerコンテナにパッケージされたNode.jsアプリケーションを使用します。Node.jsアプリケーションを作成してDockerコンテナをデプロイするには、<a href="/ja/docs/tutorials/hello-minikube/">Hello Minikubeチュートリアル</a>の指示に従ってください。</p>
```

`/tutorials/hello-minikube` は翻訳済みでページがあるので日本語ページへのリンクのままで良さそうです！

Reply:
ありがとうございます。
不慣れなもので観点が漏れてました。